rskj-core/src/main/java/org/ethereum/config/DefaultConfig.java

```diff
@@ -149,7 +141,7 @@ public MiningConfig miningConfig(RskSystemProperties rskSystemProperties) {
 
     @Bean
     public RskSystemProperties rskSystemProperties() {
-        return RskSystemProperties.CONFIG;
+        return new RskSystemProperties();
```

Comment:
This is the 🥔

Reply:
On a second though, this is still the 🥔, but we could also remove this method and just annotate the RskSystemProperties with `@Component`